src/plugins/ncr/screens/node/index.jsx

```diff
@@ -320,6 +321,24 @@ export default class AbstractNodeScreen extends React.Component {
     return <Chat key="chat" nodeRef={nodeRef} />;
   }
 
+  renderLabelsForm() {
+    const { delegate, isSaveDisabled } = this.props;
+    const node = delegate.getNode();
+
+    if (!node || !node.schema().hasMixin('gdbots:common:mixin:labelable')) {
+      return null;
+    }
+
+    return (
+      <LabelsForm
+        disabled={!isSaveDisabled}
+        key="labels"
+        node={delegate.getNode()}
+        schemas={delegate.getSchemas()}
```

Comment:
@rsumilang don't need to pass schemas

Reply:
@gdbrown I think I need to keep the schemas in ultimately for this line: https://github.com/triniti/cms-js/blob/article-labels/src/plugins/ncr/sagas/updateNodeLabelsFlow.js#L11

Reply:
@rsumilang that's fine, however, the additional node labels updated is not needed